ci/helm/preview/values.yaml

```diff
@@ -0,0 +1,143 @@
+expose:
+  Annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  config:
+    # add tls configuration
+    domain: napps.dev.nuxeo.com
+    exposer: Ingress
+    http: true
+    tlsacme: true
+    tlsSecretName: napps-tls
+    urltemplate: '"{{.Service}}-{{.Namespace}}.{{.Domain}}"'
+
+cleanup:
+  Args:
+    - --cleanup
+  Annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+
+nuxeo:
+  fullnameOverride: preview
+  nuxeo:
+    podLabels:
+      branch: "$BRANCH_NAME"
+      team: napps
+      resource: pod
+      usage: preview
+    service:
+      annotations:
+        fabric8.io/expose: "true"
+        fabric8.io/ingress.annotations: |-
+          nginx.ingress.kubernetes.io/proxy-body-size: 0
+          nginx.ingress.kubernetes.io/server-snippet: ignore_invalid_headers off;
+      labels:
+        branch: "$BRANCH_NAME"
+        team: napps
+        resource: service
+        usage: preview
+    image:
+      repository: ${DOCKER_REGISTRY}/${ORG}/nuxeo-coldstorage
+      tag: ${VERSION}
+      pullPolicy: Always
+    persistence:
+      enabled: disable
+    packages: nuxeo-web-ui-3.0.2-rc.43 amazon-s3-online-storage-$NUXEO_VERSION
```

Comment:
perhaps we can upgrade to `uxeo-web-ui-3.0.2-rc.45` wdyt

Reply:
3.0.2-rc.47 is dedicated to LTS

Reply:
No, WEB-UI LTS 2021 is not released yet, we will the `rc` but I am proposing to use the last `rc` :)

Reply:
I confess I not sure about it, also. Were you able to clarify?